libpod/image/image.go

```diff
@@ -856,24 +856,14 @@ func (i *Image) Dangling() bool {
 	return len(i.Names()) == 0
 }
 
-// Intermediate returns true if the image is cache or intermediate image.
-// Cache image has parent and child.
+// Intermediate returns true if the image is a cache or intermediate image
+// (i.e., it has children and a parent).
 func (i *Image) Intermediate(ctx context.Context) (bool, error) {
```

Comment:
Wasn’t the only caller removed as part of this PR? (And if not, I’d prefer the caller to have to supply the `layerTree` to encourage caching it over multiple calls.)